File: fib.php
#! /usr/bin/env php
<?php

  function fib($n) {
    if ($n < 2)
      return $n;
    else
      return fib($n - 1) + fib($n - 2);
  }

  $res = fib(20);
  print("$res\n");
?>

File: fib.pl
#! /usr/bin/env perl

# Calculate the n^th Fibonacci number.
sub fib {
  if ($_[0] < 2) {
    return $_[0];
  } else {
    return fib($_[0] - 1) + fib($_[0] - 2);
  }
}

$res = fib(20);
print "$res\n";
